### DISCUSSION TITLE
Is there a github sponsor in Indonesia?

### DISCUSSION THREAD

**Discussion by simpankode** [community/community#19585] [Other Feature Feedback, Questions, & Ideas] [answered]
Is there a github sponsor in Indonesia?

**Comment by prayogaea** [comment, marked as answer]
sponsored github only available in certain countries
Indonesia is not yet registered with GitHub sponsors

**Comment by melvincwng** [comment]
@simpankode, as of right now, GitHub Sponsors is not available in Indonesia.

For a list of countries where GitHub sponsors is available, please refer to this link over [here](https://github.com/sponsors).

Hope this clarifies.

Thanks, and have a great day ahead 😄. Terima kasih :)

**Comment by sinkaroid** [comment]
The sponsor website seems outdated, there is still no Indonesia in their lists, but i've received the money from the sponsors in my local bank last months, If you doubt it is available for indonesia or not, the answer is Yes
![Screenshot_586](https://user-images.githubusercontent.com/12372481/176915370-3dec84c1-7443-4b2d-a95a-d1cc321c8596.png)

**Reply by zYxDevs** [reply]
bro how is format of bri bank account number for github sponsors payouts. my payouts is always failure. its says there is something wrong on the bank account number.

**Reply by sinkaroid** [reply]
What is the issue actually, there is a bri and syariah bro, then fill it with your rekening though
![Screenshot_49](https://github.com/community/community/assets/12372481/11e2af2e-5595-44cd-8557-62b69d7be26c)

**Comment by darkterminal** [comment]
My Account have GitHub Sponsors, I am Indonesian. But I can't received payouts from Stripe Express that Integrated with My GitHub Sponsors Account.
